packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/reference/ReferenceAutocomplete.tsx

```diff
@@ -152,7 +154,8 @@ export const ReferenceAutocomplete = forwardRef(function ReferenceAutocomplete(
                     <Box padding={4}>
                       <Flex align="center" height="fill" justify="center">
                         <StyledText align="center" muted>
-                          No results for <strong>“{searchState.terms.query}”</strong>
+                          {t('navbar.new-document.no-results')}{' '}
```

Comment:
**suggestion:** consider wrapping this in a `<Trans>` component

Reply:
This was before I figured out how to use the `<Trans>` component, so yes! definitely 😄